schematools/types.py

```diff
@@ -73,19 +74,74 @@ def from_dict(cls, obj: dict):
     @property
     def tables(self) -> typing.List[DatasetTableSchema]:
         """Access the tables within the file"""
-        return [DatasetTableSchema(i, _parent_schema=self) for i in self["tables"]]
+        tables = []
+        for table_schema in self["tables"]:
+            table = DatasetTableSchema(table_schema, _parent_schema=self)
+            tables.append(table)
+        return tables
```

Comment:
Wat is de meerwaarde van dit uitschrijven? Dit doet hetzelfde als de list-comprehension.

Reply:
Oh, er was meer logica binnen de loop, niet meer.